keps/sig-instrumentation/3466-healthcheck-metrics/kep.yaml

```diff
@@ -0,0 +1,23 @@
+id: "3466"
+prnumber: ""
+name: healthcheck-metrics
+title: Kubernetes Healthcheck Metrics
+kep-number: "3466"
+authors: ['@logicalhan']
+owning-sig: sig-instrumentation
+participating-sigs: [sig-instrumentation]
+reviewers: ['@dgrisonnet', '@dashpole']
+approvers: ['@dgrisonnet']
+prr-approvers: ['@johnbelamaric']
```

Comment:
This field is no longer used - you will need to create a file in:
https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-instrumentation

BTW - I'm happy to take the PRR for it - feel free to assign to me.